### ISSUE TITLE
Sourcing R code using rpy2 doesn't work when using the package as module

### ISSUE THREAD

**Issue by faysal-ishtiaq** [opencdms/process#24]
* opencdms_process version: 0.1.0
* Python version: 3.8
* Operating System: Pop OS

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

I wrote a class where I used `windrose` function from `opencdms_process.process.climatol`

```
if __name__ == "__main__":

    filters = {
        'src_id': 838,
        'period': 'hourly',
        'year': 1991,
        'elements': ['wind_speed', 'wind_direction'],
    }

    windrose_chart = WindroseDataProcessor(filters) \
        .generate_chart(base64_encoded=True)

    print(windrose_chart)
```
when I run the file from terminal like `python file_to_run.py` with the code above, it runs successfully. But when I import it as module, the following error occurs:

```
Traceback (most recent call last):
File "/usr/lib/python3.8/code.py", line 90, in runcode
exec(code, self.locals)
File "<input>", line 8, in <module>
File "<input>", line 16, in generate_chart
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/opencdms_process-0.1.0-py3.8.egg/opencdms_process/process/climatol/__init__.py", line 27, in windrose
r(f"""
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/rpy2/robjects/__init__.py", line 438, in __call__
res = self.eval(p)
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/rpy2/robjects/functions.py", line 198, in __call__
return (super(SignatureTranslatedFunction, self)
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/rpy2/robjects/functions.py", line 125, in __call__
res = super(Function, self).__call__(*new_args, **new_kwargs)
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/rpy2/rinterface_lib/conversion.py", line 45, in _
cdata = function(*args, **kwargs)
File "/home/faysal/PycharmProjects/opencdms-process/venv/lib/python3.8/site-packages/rpy2/rinterface.py", line 680, in __call__
raise embedded.RRuntimeError(_rinterface._geterrmessage())
rpy2.rinterface_lib.embedded.RRuntimeError: Error in file(filename, "r", encoding = encoding) :
cannot open the connection
```

**Comment by isedwards** [comment]
Is there a chance this is just saying 'file not found'?

Could you try copying the script to `/tmp/` and giving this path when sourcing the script?

Or try `print(script)` to see if there's any part of the path R may not like?

**Comment by faysal-ishtiaq** [comment]
it could be. I will look into it.

**Comment by isedwards** [comment]
Resolved in [3ff32f7b](https://github.com/opencdms/opencdms-process/pull/23/commits/3ff32f7b124d68b90654c0a2cd256e3a6aa010ba)
